lib/sqs.test.ts

```diff
@@ -1,20 +1,21 @@
 import test from "ava";
-import SQS from "aws-sdk/clients/sqs";
-import { AbortController } from "node-abort-controller";
+import { SQSClient } from "@aws-sdk/client-sqs";
 import { ulid } from "ulid";
 
-import { SQSData, SQSHelper } from "./sqs";
+import { SQS, SQSData, SQSHelper } from "./sqs";
```

Comment:
why are we importing SQS from sqs instead of aws/client-sqs now?

Reply:
We're importing the type, which is a stripped down version of the SQS client (it just has send), so that we can implement a mock for it for testing.

Reply:
I'm not following, what do you mean?

Reply:
AWS SQSClient has a bunch of methods to do things that we don't use, so in sqs.ts we define a type (SQS) that has just the method we use (`send()`), this way we can create our own fake compatible client that we can substitute for the real thing when running tests, without the extra work of adding the unused methods.